src/clusterfuzz/_internal/protos/uworker_msg.proto

```diff
@@ -114,6 +114,11 @@ message Input {
   optional string module_name = 18;
   optional google.protobuf.Timestamp preprocess_start_time = 19;
   repeated string global_blacklisted_functions = 20;
+  optional TestCaseManagerInput testcase_manager_input = 21;
+}
+
+message TestCaseManagerInput {
+  optional google.datastore.v1.Entity fuzz_target = 1;
```

Comment:
Would it make sense to consolidate the `fuzz_target` here with what's in FuzzTaskInput? 

e.g. either `FuzzTaskInput` would use `TestCaseManagerInput`, or we move `fuzz_target` up into `Input` ?

Reply:
I think hoisting it into Input makes sense.